From stable-bounces@linux.kernel.org Wed Apr  5 12:01:43 2006
Date: Wed, 5 Apr 2006 12:03:45 -0700
From: "Randy.Dunlap" <rdunlap@xenotime.net>
To: stable@kernel.org, lkml <linux-kernel@vger.kernel.org>
Message-Id: <20060405120345.6ad380de.rdunlap@xenotime.net>
Cc: 
Subject: isd200: limit to BLK_DEV_IDE


From: Randy Dunlap <rdunlap@xenotime.net>

Limit USB_STORAGE_ISD200 to whatever BLK_DEV_IDE and USB_STORAGE
are set to (y, m) since isd200 calls ide_fix_driveid() in the
BLK_DEV_IDE code.

Signed-off-by: Randy Dunlap <rdunlap@xenotime.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/Kconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- linux-2.6.16.1.orig/drivers/usb/storage/Kconfig
+++ linux-2.6.16.1/drivers/usb/storage/Kconfig
@@ -48,7 +48,8 @@ config USB_STORAGE_FREECOM
 
 config USB_STORAGE_ISD200
 	bool "ISD-200 USB/ATA Bridge support"
-	depends on USB_STORAGE && BLK_DEV_IDE
+	depends on USB_STORAGE
+	depends on BLK_DEV_IDE=y || BLK_DEV_IDE=USB_STORAGE
 	---help---
 	  Say Y here if you want to use USB Mass Store devices based
 	  on the In-Systems Design ISD-200 USB/ATA bridge.
